coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,19 @@
-class InvalidFilterException(Exception):
-    pass
+class InvalidFilterException(BaseException):
+
+    def __init__(self, filter_name, available_filters):
+        self.filter = filter_name
+        self.available_filters = list(available_filters.split(', '))
+        if self.available_filters is None:
+            message = 'Invalid Filter'
+
+        if self.available_filters is not None:
+            message = 'Invalid filter. Available filters are : ' + \
+              str(self.available_filters)
+
+        else:
+            message = self.filter + \
+                      ' is an invalid filter. Available filters: ' + \
+                      str(self.available_filters)
```

Comment:
@shreyans800755 thought not to use different conditions, so we have always the same message and require the user of this exception to pass always the filter-name and available-filters. Both is fine with me, although @shreyans800755's variant is easier to implement.